rust/vapid/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "vapid"
+version = "0.1.0"
+authors = ["jrconlin <jconlin+git@mozilla.com>"]
+
+[dependencies]
+openssl = "0.9.16"
+serde_json = "1.0.2"
+base64 = "0.6.0"
+time = "0.1"
+asn1 = "*"
```

Comment:
In general for "library crates" (ones that you depend on, not final end-user binaries and such) it's generally considered an antipattern to depend on `*` as a version constraint because it means that when `asn1`, for example, publishes a new major version with API breakage it could accidentally also break users of the `vapid` crate when they run `cargo update`.

We're still working on a `cargo add asn1` (or the like) to auto-add this to `Cargo.toml` so you don't have to worry about the version numbers yourself, we're just not quite there yet unfortunately :(. The progress today is in [`cargo-edit`](https://crates.io/crates/cargo-edit)

Reply:
Cool. I've later realized I don't need asn1 right now, since I'm hand-rolling the asn1 conversions. I definitely agree that not version locking is a pretty scary idea, so I'll happily drop this.